debs/SPECS/3.11.0/wazuh-manager/debian/postrm

```diff
@@ -27,7 +27,7 @@ case "$1" in
 
 
 
-    if [ -d /run/systemd/system ] ; then
+    if [ ! -z "$(file /sbin/init |grep "systemd")" ]; then
```

Comment:
Same https://github.com/wazuh/wazuh-packages/pull/269/files#r370579928

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/269/commits/6765088880328ba825528bf1164578088b3bf8fa